bun/whatsapp-with-vonage/src/main.ts

```diff
@@ -0,0 +1,65 @@
+import jwt from "jsonwebtoken";
+import { fetch } from "undici";
+import { getStaticFiles } from "./utils";
+
+export default async ({ req, res, log, error }) => {
```

Comment:
We need to throw if the environment variables are not set like here https://github.com/appwrite/templates/blob/6dab35f1f221ba075640d93f5a8e374f4fba4349/node/whatsapp-with-vonage/src/main.js#L7